bbc/simorgh
# Gnadtech 2349 2

Resolves GNADTECH-2349

**Overall change:**
Add Neilsen analytics to story/article AMP pages in Australia

**Code changes:**

Created a NielsenAnalytics component that writes some JSON to the page if on an AMP page  and nielsenAnalytics toggle is enabled.

**Testing:**

- Check that values are only sent to Neilsen on AMP pages when in Australia
- Check correct values are sent (see jira ticket above)

The below will result in calls to URLs containing **imrworldwide** along with other bits of metadata added by Neilsen themselves....

----

**Comments from Stephen Macleod:**

Geo limited to Australia
Pages limited to text (so do not need to fire on video events on embedded players)
If we can, set the reserved variables below with the same values as on web pages
 
**Required Variables and Values**

APID  (note this will also serve as the production ID once validated by Nielsen) 

News 474C2B0B-0C04-4182-BCFB-FC9469A48C9B
Sport DC4AFDB2-B352-4D51-81EF-38BE41114F22
 

Below are examples of browser static metadata objects that would need to be created for each page and called along with the 'staticstart' function in order to generate a page view -

https://www.google.co.uk/amp/s/www.bbc.com/news/business-58007120.amp

var staticMetadataObject = { 
  type: "static", //Required: the same for every static asset 
  section: "BBC News Business", //Required
  segC: "BBC – Google AMP"
};

Static metadata example for - https://www.bbc.com/sport/formula1/51607740 

var staticMetadataObject = { 
  type: "static", //Required: the same for every static asset 
  section: "BBC Sport Formula 1", //Required
  segC: "BBC – Google AMP"
};

---

- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [x] This PR requires manual testing (to be done by BBC Studios/GNL)
